openfoam-interface/libs/sourceFiles/LESModel.C

```diff
@@ -0,0 +1,207 @@
+/*---------------------------------------------------------------------------*\
```

Comment:
Another general question is it looks like there are some OpenFoam source files now in the repository.  For example, openfoam-interface/libs/sourceFiles/LESModel.C  is the exact same as "OpenFOAM-XXX/src/turbulenceModels/incompressible/LES/LESModel/LESModel.C" in OpenFoam (2.2.x).  At least some of the files are unchanged from OpenFoam.  Is there a specific reason for this?  Could the OpenFoam library/src instead be used?